generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.html.ejs

```diff
@@ -49,39 +49,39 @@
                         <input type="password" class="form-control" id="newPassword" name="newPassword"
                                placeholder="{{'global.form.newpassword.placeholder' | translate}}"
                                formControlName="newPassword" #newPassword>
-                        <div *ngIf="passwordForm.get('newPassword').invalid && (passwordForm.get('newPassword').dirty || passwordForm.get('newPassword').touched)">
+                        <div *ngIf="passwordForm.get('newPassword')?.invalid && (passwordForm.get('newPassword')?.dirty || passwordForm.get('newPassword')?.touched)">
```

Comment:
Are those fields really nullable ? I don't see the case where `passwordForm.get('newPassword')` could be undefined

Reply:
I accidentally answered in separate comment instead of this place.